xscape/plotcostsAnalytic.py

```diff
@@ -1,6 +1,7 @@
 # plotcostsAnalytic.py
```

Comment:
It seems that you are also modifying the `plotcostsAnalysic.py` file here. I think you want to modify this file but not add a new file?